README.mkdn

```diff
@@ -260,7 +260,16 @@ $ /usr/local/lpcscrypt/bin/lpcscrypt clockslow
 $
 ```
 
-4. Run `lpcscrypt program +c <path-to-lpc4322_bl_crc.bin> BankA`:
+4. Run `lpcscrypt program +w1 0xAA BankA` to overwrite existing firmware
```

Comment:
This is different than the command given in meta which is what I've been using `${LPCSCRYPT_ROOT}/bin/lpcscrypt program +w1 0x0 BANKA`
I'm guessing this is just wiping the bank so it doesn't matter what value is used?

Reply:
It shouldn't matter but 0x0 is probably less confusing.  Ideally `lpcscrypt eraseFlash BankA` would work but it doesn't so this is the hack instead.